### PR TITLE
Bootable JAR 7.0.0.Final doc

### PR DESCRIPTION
**Pull request by jfdenise**

### CODE REVIEW

**Comment by jfdenise** [comment]
@jamezp @bstansberry , Bootable 7 doc update. Expects 26 doc directory to be created.

**Review by jamezp — APPROVED** [review]
This looks good, but can't be merged until after the WildFly 26 documentation is merged.

**Comment by bstansberry** [comment]
The 26 doc URLs will work now that #41 is merged but I'll hold off on merging this until I update the overall root index.html etc to reference the 26 docs.

**Pull request merged**
